packages/toast-ui.grid/docs/en/pagination.md

```diff
@@ -26,7 +26,7 @@ const grid = new Grid({
 });
 ```
 
-As you can see in the example above, in order to sync remote data operations, just define how many pages to show in one page with the `pageOptions.perPage` and determine the outlines of request and response data with Data Source. You can view related information [here](https://github.com/nhn/tui.grid/blob/master/packages/toast-ui.grid/docs/en/data-source.md) in greater detail.
+As you can see in the example above, in order to sync remote data operations, just define how many rows or elements to show in one page with the `pageOptions.perPage` and adapt the request and response parameters with the Data Source. You can view related information [here](https://github.com/nhn/tui.grid/blob/master/packages/toast-ui.grid/docs/en/data-source.md) for more information.
```

Comment:
@joebordes 
Thank you for PR!
I think that It's better to modify a few sentence. Could you fix it?
1. the meaning of `rows` is fit to this sentence than `elements`.
**just define how many rows or elements to show** => **just define how many rows to show**
2. **the Data Source** => **Data Source**

Reply:
Done, but I do not agree with eliminating the "the", nor does Grammarly

Reply:
@joebordes 
First of all, this PR will be merged.
We discuss about this issue(add `the`) you commented internally and apply the result to other tutorials.